docs/Migration-Guide.md

```diff
@@ -0,0 +1,90 @@
+# Migration Guide
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Migrating from v1 to v2](#migrating-from-v1-to-v2)
+  - [1. Integration Service changes](#1-integration-service-changes)
+  - [2. Payone portal changes](#2-payone-portal-changes)
+  - [3. Changes in the shops which uses the service](#3-changes-in-the-shops-which-uses-the-service)
+  - [4. Setup a new tenant (branch,shop,merchant)](#4-setup-a-new-tenant-branchshopmerchant)
+  - [5. PROFIT](#5-profit)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Migrating from v1 to v2
+
+Version 2 introduced multi-tenancy feature. This is a ***breaking-change*** thus requires additional steps in the 
+service and Payone provider setup.
+
+**Note**: after version 2 release the first version likely will be abandoned hence we recommend to do the migration as 
+soon as it is possible for your project.
+
+If you are setting up the project from scratch - likely you should just follow [Multitenancy](/README.md#multitenancy) 
+section in the main documentation page.
+
+### 1. Integration Service changes
+
+1. One mandatory property added: `TENANTS`, a comma (or semicolon) separate list of your tenants (branches, franchising etc).
```

Comment:
it is done just 5 lines below: https://github.com/commercetools/commercetools-payone-integration/blob/master/docs/Migration-Guide.md#1-integration-service-changes